recipe_scrapers/tineno.py

```diff
@@ -1,97 +1,93 @@
+import re
+from typing import Optional, Tuple, Union
+
 from ._abstract import AbstractScraper
-from ._utils import get_minutes, get_yields, normalize_string
+from ._utils import change_keys
+
+
+def fix_json_ld_property_keys(k):
+    if k == "ItemListElement":
+        return "itemListElement"
+    return k
 
 
 class TineNo(AbstractScraper):
+    def __init__(
+        self,
+        url,
+        proxies: Optional[str] = None,
+        timeout: Optional[Union[float, Tuple, None]] = None,
+        wild_mode: Optional[bool] = False,
+    ):
+        super().__init__(url, proxies, timeout, wild_mode)
+        # Fix incorrect property naming
+        self.schema.data = change_keys(self.schema.data, fix_json_ld_property_keys)
+
     @classmethod
     def host(cls):
         return "tine.no"
 
+    def author(self):
+        return self.schema.author()
+
     def title(self):
-        return self.soup.find("h1").get_text()
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
 
     def total_time(self):
-        total_time = 0
-        tt1 = self.soup.find("li", {"class": "m-recipe-overview__total-time"})
-        if tt1:
-            tt = tt1.find("span", {"class": "t-info"})
-            tt1 = normalize_string(tt.get_text())
-            tt2 = get_minutes(tt1)
-            if tt1 and (tt2 == 0):
-                total_time = tt1
-            else:
-                total_time = tt2
-        return total_time
+        return self.schema.total_time()
 
     def yields(self):
-        recipe_yield = self.soup.find("input", {"id": "portions"})
-        if recipe_yield:
-            return recipe_yield["value"]
-        else:
-            return get_yields(
-                self.soup.find(
-                    "div", {"class": "recipe-adjust-servings__original-serving"}
-                ).get_text()
-            )
+        return self.schema.yields()
 
     def image(self):
         image = self.soup.find("img", {"id": "HeaderMediaContent"})
         if image:
```

Comment:
Optional suggestion: we could negate this conditional and return early (`return None`) when no `image` value is found?

That would allow a reduction in the level of indentation for the rest of the code contained in the method.

Reply:
Absolutely, will fix!

Reply:
Looks like the same could be done for `srcset_str`